### PR TITLE
Agent tutorial and minor update to conceptual guide

### PR DESCRIPTION
**Pull request by rlancemartin**

### CODE REVIEW

File: versioned_docs/version-2.0/tutorials/Developers/agents.mdx [hunk 1]
```diff
@@ -0,0 +1,507 @@
+---
+sidebar_position: 6
+---
+
+# Agent Evaluations
```
Comment (agola11) [inline versioned_docs/version-2.0/tutorials/Developers/agents.mdx:5]: rename to "Evaluate an agent"

Comment (agola11) [reply versioned_docs/version-2.0/tutorials/Developers/agents.mdx:5]: also, add to index.md for tutorials

File: versioned_docs/version-2.0/tutorials/Developers/agents.mdx [hunk 2]
```diff
@@ -0,0 +1,507 @@
...
+
+We will walk through the evaluation workflow for RAG (retrieval augmented generation).
```
Comment (agola11) [inline versioned_docs/version-2.0/tutorials/Developers/agents.mdx:7]: Agent, not RAG. Also discuss the agent we are building, eg the SQL agent. Explain the tools we are giving it

File: versioned_docs/version-2.0/tutorials/Developers/agents.mdx [hunk 3]
```diff
@@ -0,0 +1,507 @@
...
+
+First, we ensure that the necessary packages are installed.
+
+```python
+%pip install --upgrade --quiet  langchain langchain-community langchain-experimental langchain_openai langgraph
```
Comment (agola11) [inline versioned_docs/version-2.0/tutorials/Developers/agents.mdx:12]: are these deps correct? We never use experimental

**Review by agola11 — COMMENTED** [review]
add links to reference the langgraph docs for more info

**Comment by rlancemartin** [comment]
All comments should be resolved

**Review by agola11 — APPROVED** [review]

**Pull request merged**